dahdi-tools not compiling on GCC10

Maintainer: @VittGam 
Environment: master +  https://github.com/openwrt/openwrt/pull/3126

```
OpenWrt-libtool: link: mips-openwrt-linux-musl-gcc -I. -I./xtalk/include -Wall -Os -pipe -mno-branch-likely -mips32r2 -mtune=74kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -mips16 -minterlink-mips16 -ffile-prefix-map=/home/mangix/devstuff/openwrt/build_dir/target-mips_74kc+dsp2_musl/dahdi-tools-3.0.0=dahdi-tools-3.0.0 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z -Wl,now -Wl,-z -Wl,relro -znow -zrelro -o astribank_tool astribank_tool-astribank_tool.o  -L/home/mangix/devstuff/openwrt/staging_dir/target-mips_74kc+dsp2_musl/usr/lib -L/home/mangix/devstuff/openwrt/staging_dir/target-mips_74kc+dsp2_musl/lib -L/home/mangix/devstuff/openwrt/staging_dir/toolchain-mips_74kc+dsp2_gcc-10.1.0_musl/usr/lib -L/home/mangix/devstuff/openwrt/staging_dir/toolchain-mips_74kc+dsp2_gcc-10.1.0_musl/lib ./.libs/libastribank.a -lusb-1.0 -lpthread -lm
/home/mangix/devstuff/openwrt/staging_dir/toolchain-mips_74kc+dsp2_gcc-10.1.0_musl/lib/gcc/mips-openwrt-linux-musl/10.1.0/../../../../mips-openwrt-linux-musl/bin/ld: ./.libs/libastribank.a(libxtalk_la-xtalk_sync.o):(.bss+0x0): multiple definition of `members'; ./.libs/libastribank.a(libastribank_la-mpptalk.o):(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
```

Build failure is identical to passing

TARGET_CFLAGS += -fno-common

to previous GCC versions.

--- a/libs/dahdi-tools/Makefile
+++ b/libs/dahdi-tools/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dahdi-tools
 PKG_VERSION:=3.0.0
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.asterisk.org/pub/telephony/dahdi-tools/releases
@@ -53,6 +53,9 @@ define Package/dahdi-tools-libtonezone
 endef
 
 CONFIGURE_ARGS+=--disable-silent-rules
+
+# https://issues.asterisk.org/jira/browse/DAHTOOL-85
+TARGET_CFLAGS+=-fcommon
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/lib

